types/intercom-client/intercom-client-tests.ts

```diff
@@ -1,3 +1,3 @@
 import * as intercom from "intercom-client";
```

Comment:
Could you please add a couple of tests for the new definitions?

Reply:
@armanio123 I can work on getting those in over the next week. Thanks for the review